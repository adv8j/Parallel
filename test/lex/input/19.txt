// \n this should ideally only give int, token and ; as tokens
int token;
