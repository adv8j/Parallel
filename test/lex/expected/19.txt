single line comment
INT
IDENTIFIER
SEMICOLON
